lib/core/device_drivers.rb

```diff
@@ -179,22 +179,19 @@ def merge_chrome_ops(config_caps, case_caps)
 
   # build the Chrome driver, given a set of options
   def build_chrome_driver(chrome_ops)
+    # local selenium instance
+    chromeOptions = Selenium::WebDriver::Remote::Capabilities.chrome(
+      "goog:chromeOptions": chrome_ops,
+    )
     if @url.nil?
-      # local selenium instance
-      localChromeOptions = Selenium::WebDriver::Chrome::Options.new(
-        options: chrome_ops,
-      )
       driver = Selenium::WebDriver.for(
-        :chrome, options: localChromeOptions
+        :chrome, capabilities: chromeOptions
```

Comment:
Not sure why this is `capabilities` instead of `options`

Reply:
my mistake, fixed